etc/di.xml

```diff
@@ -14,4 +14,21 @@
     <type name="Magento\Framework\App\Request\CsrfValidator">
         <plugin name="csrf_validator_skip" type="Razorpay\Magento\Plugin\CsrfValidatorSkip" />
     </type>
+    <virtualType name="RazorpayLogger" type="Magento\Framework\Logger\Monolog">
+	    <arguments>
+	        <argument name="handlers" xsi:type="array">
+	            <item name="error" xsi:type="object">Razorpay\Magento\Model\LogHandler</item>
+	        </argument>
+	    </arguments>
+	</virtualType>
+	<type name="Razorpay\Magento\Controller\Payment\Order">
```

Comment:
@enishant , we must add logging for other controllers and related files , like model observers etc if required.

Reply:
@enishant , any updates on this

Reply:
@ramth05 Added logging to related controllers & model